src/main/java/prefs/Preferences.java

```diff
@@ -138,32 +202,51 @@ public void setLastOpenBoard(String board) {
         
         return getLastOpenBoard();
     }
-    
+
+    /**
+     * Clears the last open board
+     */
     public void clearLastOpenBoard() {
-        global.clearLastOpenBoard();
+        sessionConfig.clearLastOpenBoard();
     }
-    
+
+    /**
+     * Retrieves the panels of the specific board.
+     * @param board The name of the board to retrieve the panels from
+     * @return A list of panel infos from that board
+     */
     public List<PanelInfo> getBoardPanels(String board) {
```

Comment:
While we are on it, let's rename the variable to `boardName`. The comment can be `Retrieves the panels of a board`